clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/input/kafka/ttl/VeniceRmdTTLFilter.java

```diff
@@ -48,13 +50,11 @@ public abstract class VeniceRmdTTLFilter<INPUT_VALUE> extends AbstractVeniceFilt
 
   public VeniceRmdTTLFilter(final VeniceProperties props) throws IOException {
     super();
-    ttlPolicy = TTLResolutionPolicy.valueOf(props.getInt(VenicePushJob.REPUSH_TTL_POLICY));
-    long ttlInMs = TimeUnit.SECONDS.toMillis(props.getLong(VenicePushJob.REPUSH_TTL_IN_SECONDS));
-    long ttlStartTimestamp = props.getLong(VenicePushJob.REPUSH_TTL_START_TIMESTAMP);
-    this.filterTimestamp = ttlStartTimestamp - ttlInMs - 1;
-    this.schemaSource = new HDFSSchemaSource(
-        props.getString(VenicePushJob.VALUE_SCHEMA_DIR),
-        props.getString(VenicePushJob.RMD_SCHEMA_DIR));
+    ttlPolicy = TTLResolutionPolicy.valueOf(props.getInt(REPUSH_TTL_POLICY));
+    long ttlStartTimestamp = props.getLong(REPUSH_TTL_START_TIMESTAMP);
+    // Filter anything that is before the start timestamp.
+    this.filterTimestamp = ttlStartTimestamp - 1;
```

Comment:
This is not correct right? filter timestamp = TTL job start TS - rewind time in MS - 1.

Reply:
I modified the variables and constants from what was used before. `REPUSH_TTL_START_TIMESTAMP` is set as `TTL job start TS - rewind time in MS`